pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -542,3 +543,6 @@ The users can pull the data for a given 24 hour time interval date range from to
 - For Analytics API, Pagination is not needed.
 - For Proactive Campaign API, Pagination is not supported yet.
 - For Proactive Campaign Analytics API, Pagination is not needed.
+
+<strong>11. Is reporting data real time ? What is the delay in reporting data from the time proactive campaigns or C2M deflections are created ?</strong>
```

Comment:
nit: just use consistent casing for Proactive Messaging campaign instead of "proactive campaigns"

Reply:
replace "Is reporting data" with "Is Outbound Reporting API real time?"

Reply:
sure

Reply:
Done.